### PR TITLE
Update Metadata

### PR DESCRIPTION
**Pull request by galgalesh**
Update the metadata of the Signal Snap.

### CODE REVIEW

**Review by hellsworth — APPROVED** [review]

**Comment by hellsworth** [comment]
Lgtm!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+```
+
+([Don't have snapd installed?](https://snapcraft.io/docs/core/install))
+
+<p align="center">Published for <img src="https://raw.githubusercontent.com/anythingcodes/slack-emoji-for-techies/gh-pages/emoji/tux.png" align="top" width="24" /> with :gift_heart: by Snapcrafters</p>
+
+## License
+
+* The license of the build files in this repository is GNU Affero General Public License v3.0 only
+* Signal Desktop itself has [its own license](https://github.com/signalapp/Signal-Desktop/blob/main/LICENSE).
```
Comment (diddledani) [inline README.md:26]: It looks like Signal Desktop uses the GNU AGPL 3.0 only, too so you can get away with dropping this line, maybe?

**Review by diddledani — APPROVED** [review]
LGTM with one comment

File: README.md (hunk 1 above)
Comment (galgalesh) [reply README.md:26]: Agreed, I'll drop it in a following PR.

**Pull request merged**